search/ranker.cpp

```diff
@@ -391,21 +430,22 @@ void Ranker::UpdateResults(bool lastUpdate)
 
     BailIfCancelled();
 
+    auto const & rankerResult = m_tentativeResults[i];
+
     if (m_params.m_viewportSearch)
     {
       // Viewport results don't need calculated address.
-      Result res = GenerateFinalResult(m_tentativeResults[i], false /* needAddress */);
+      Result res = MakeResult(rankerResult, false /* needAddress */, false /* needHighlighting */);
```

Comment:
Could you please rename "res" to "result"?

Reply:
done